### ISSUE TITLE
Attention problem

### ISSUE THREAD

**Issue by ttslr** [shtoshni92/g2p#3]
Excuse me!
When will you release your   code about "attention mechanism"?

Thank you very much!

**Comment by shtoshni** [comment]
Sorry I haven't released that part of the code but if you might have read my paper, attention doesn't add much to performance. In case you're really keen on adding attention, I have other projects which do it (speech-parsing, e2e_asr).

**Comment by abhishek-niranjan** [comment]
@shtoshni92 I was going through the paper and found that the Enc-Dec with global attention outperforms other architectures. I'm not really sure about what you mean that attention didn't add much to performance. 
Can you please explain? 
Also, I couldn't reproduce the results stated in the paper, maybe I'm missing out on setting the parameters but the PER seems way off from what reported in the paper. 
Thanks.

**Comment by shtoshni** [comment]
The performance difference is quite small because the sequences are not too big (25-30 long sequences at best). I have updated the code and it has the attention and the default params now: @ttslr and @abhishek-niranjan. I did run an experiment for 50 epochs and got a WER of ~22.08 and PER of ~5.08. By 100 epochs, I expect it to improve a little more. Let me know if you guys can reproduce the results.
